docs/terminal/sydney.md

```diff
@@ -227,36 +236,45 @@ The divisions of the airspace between **SAN**, **SAS**, **SDS**, **SDN**, **SFW*
 !!! note
     The following diagrams do not include non SY TCU areas of responsibility such as BK CTR or CN CTR
 
+### Side Profile
+<figure markdown>
+![SY TCU Side Profile](img/syside.png){ width="700" }
+  <figcaption>SY TCU Side Profile</figcaption>
+</figure>
+
 ### 07
 <figure markdown>
-![07 TCU Structure](img/SY07annotated.png){ width="700" }
+![07 TCU Structure](img/sy07annotated.png){ width="700" }
   <figcaption>07 TCU Structure</figcaption>
 </figure>
 
 ### 16 PROPS
 <figure markdown>
-![16 PROPS TCU Structure](img/SY16PROPSannotated.png){ width="700" }
+![16 PROPS TCU Structure](img/sy16PROPSannotated.png){ width="700" }
   <figcaption>16 PROPS TCU Structure</figcaption>
 </figure>
 
 ### 25
 <figure markdown>
-![25 TCU Structure](img/SY25annotated.png){ width="700" }
+![25 TCU Structure](img/sy25annotated.png){ width="700" }
   <figcaption>25 TCU Structure</figcaption>
 </figure>
 
 ### 34 PROPS
 <figure markdown>
-![34 PROPS TCU Structure](img/SY34PROPSannotated.png){ width="700" }
+![34 PROPS TCU Structure](img/sy34PROPSannotated.png){ width="700" }
   <figcaption>34 PROPS TCU Structure</figcaption>
 </figure>
 
 ### SODPROPS
 <figure markdown>
-![SODPROPS TCU Structure](img/SYSODPROPSannotated.png){ width="700" }
+![SODPROPS TCU Structure](img/sySODPROPSannotated.png){ width="700" }
   <figcaption>SODPROPS TCU Structure</figcaption>
 </figure>
 
+!!! note
+    SODPROPS runway mode may not be used when Director is online, as they do not own any airspace in this mode.
```

Comment:
Is this correct? 
Or does director own SFW, whilst SFE is released to SDS?

Reply:
we dont have any SFW or SFE on the airspace diagrams, not shown in the labels anywhere, but havent yet confirmed with SME

Reply:
also for vatsim purposes, would be fairly counter intuitive to have a director spot open during a low capacity runway mode

Reply:
Wouldn't it just be easier to say 'Director positions unavailable during SODPROPs due to...'

As it currently is it makes it sound like controllers shouldn't use SODPROPs at all.

Reply:
i thought about that, but i also dont want to make it sound like you can log on as director when it gets busy. we want to make sure people arent continuing to use sodprops in the middle of a busy milk run when we need the capacity of the parallel runways. director is online for on average, about 1 hour a week, it hardly precludes the use of SODPROPS

Reply:
What's the AAR of SODPROPS mode?

Reply:
cant remember exactly but i remember it being the lowest when i checked it

Reply:
SME has confirmed there is director for SODPROPS, ill make some changes